cmd/plugin/krew.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ingress-nginx
+spec:
+  shortDescription: Interact with ingress-nginx
+  description: |
+    The official kubectl plugin for ingress-nginx.
+  version: {{ .TagName }}
+  homepage: https://kubernetes.github.io/ingress-nginx/kubectl-plugin/
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    {{addURIAndSha "https://github.com/kubernetes/ingress-nginx/releases/download/{{ .TagName }}/kubectl-ingress-nginx_darwin_arm64.tar.gz" .TagName }}
+    bin: kubectl-ingress-nginx
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    {{addURIAndSha "https://github.com/kubernetes/ingress-nginx/releases/download/{{ .TagName }}/kubectl-ingress-nginx_darwin_amd64.tar.gz" .TagName }}
+    bin: kubectl-ingress-nginx
+    selector:
```

Comment:
@Volatus I noticed this small syntax error in the krew.yaml
it's missing a leading `-` on the linux/amd64 entry at https://github.com/kubernetes/ingress-nginx/blob/7e2e70135e751b254b8d8459e66d0198fd34e5b3/cmd/plugin/krew.yaml#L24